cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/dslReferenceContent.groovy

```diff
@@ -0,0 +1,151 @@
+package org.jenkinsci.plugins.workflow.cps.Snippetizer
+
+import org.jenkinsci.plugins.workflow.cps.GlobalVariable
+import org.jenkinsci.plugins.workflow.cps.Snippetizer
+import org.jenkinsci.plugins.workflow.steps.StepDescriptor
+import org.jenkinsci.plugins.workflow.structs.DescribableHelper
+
+import javax.servlet.RequestDispatcher
+
+Snippetizer snippetizer = my;
+
+def l = namespace(lib.LayoutTagLib)
+def st = namespace("jelly:stapler")
+
+
+h1(_("Steps"))
+for (StepDescriptor d : snippetizer.getStepDescriptors(false)) {
```

Comment:
So my thinking is that I'd rather have the limitations of Groovy views over trying to generate Groovy and HTML via `String`s in Java. But in the end, I'll defer to you - I just couldn't find a way to do the GDSL/DSLD in particular that I felt comfortable with, and I felt *much* more comfortable doing the HTML in the view as well. *shrug*